src/platform/user/profile/vet360/containers/VAPProfileField.jsx

```diff
@@ -74,10 +74,12 @@ class VAPProfileField extends React.Component {
 
   static defaultProps = {
     fieldName: '',
+    showConfirmCancelModal: false,
```

Comment:
`hasUnsavedEdits`?

Reply:
sorry, I mean should the entry in `defaultProps` be `hasUnsavedEdits`?

Reply:
Haha, yes. I need to make myself some coffee ☕